sdk/communication/azure-communication-callingserver/azure/communication/callingserver/aio/_call_connection_async.py

```diff
@@ -76,16 +77,32 @@ async def play_audio(
             play_audio_options: 'PlayAudioOptions',
             **kwargs: Any
         ) -> PlayAudioResult:
-
+        
         if not audio_file_uri:
             raise ValueError("audio_file_uri can not be None")
 
         if not play_audio_options:
             raise ValueError("options can not be None")
 
+        try:
+            if not audio_file_uri.lower().startswith('http'):
```

Comment:
lets avoid modifying user input. just use a validator to validate the url and raise error